src/encoding/text.rs

```diff
@@ -742,6 +746,24 @@ mod tests {
         parse_with_python_client(encoded);
     }
 
+    #[test]
+    fn encode_histogram_family_with_empty_family_labels() {
+        let mut registry = Registry::default();
+        let family =
+            Family::new_with_constructor(|| Histogram::new(exponential_buckets(1.0, 2.0, 10)));
+        registry.register("my_histogram", "My histogram", family.clone());
```

Comment:
Just to make sure we are on the same page, you don't need to wrap a `Histogram` in a `Family`.

```suggestion
        let histogram = Histogram::new(exponential_buckets(1.0, 2.0, 10));
        registry.register("my_histogram", "My histogram", historgam.clone());
```

This should fix the encoding failure, correct?

Reply:
Ah - this makes more sense now. I didn't realize that the `Family` wrapper was optional. Using the histogram directly indeed does fix the encoding failure.